67_RayQueryGeometry/common.hlsl

```diff
@@ -0,0 +1,25 @@
+#ifndef RQG_COMMON_HLSL
+#define RQG_COMMON_HLSL
+
+NBL_CONSTEXPR uint32_t WorkgroupSize = 16;
+
+#ifdef __HLSL_VERSION
+struct VertexData
+{
+    float3 position;
+    float4 color;
+    float2 uv;
+    float3 normal;
+};
+
+struct SCameraParameters //! matches CPU version size & alignment (160, 4)
+{
+    float3 camPos;
+
+	float4x4 MVP;
+	float3x4 V;
+    float3x4 P;
+};
```

Comment:
this struct can be shared between HLSL and C++ as long as you `using namespace nbl::hlsl` beforehand

Reply:
still a TODO